packages/@aws-cdk/aws-ecs/lib/log-drivers/log-drivers.ts

```diff
@@ -59,4 +60,11 @@ export class LogDrivers {
   public static syslog(props?: SyslogLogDriverProps): LogDriver {
     return new SyslogLogDriver(props);
   }
+
+  /**
+   * Creates a log driver configuration that sends log information to firelens log router.
```

Comment:
Can you add a link to firelens examples? Since this configuration is so variable and friable, we should provide useful context in the documentation strings. 
https://github.com/aws-samples/amazon-ecs-firelens-examples

Reply:
thanks, added the link in latest commit.